gandalf.js

```diff
@@ -8,126 +8,125 @@
  *
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?